content/blog/2022-07-15-july-22-heartbeat.md

```diff
@@ -0,0 +1,373 @@
+
+![Iterative Community shout out from Alex Strick van Linshoten](/uploads/images/2022-07-15/alex-strick-van-linshoten.png '=800')
+_Thanks for the shout-out Alex!
+([Source link](https://mlops.systems/tools/redactionmodel/computervision/mlops/2022/05/24/data-versioning-dvc.html#-appendix-how-to-switch-from-git-lfs-to-dvc))_
+
+### MyMLOps Stack
+
+[MyMLOps.com](https://mymlops.com/) provides a tool to help you build a cool
+diagram for your MLOps Stack. There's no about page there or indication of who
+made this for the greater MLOps Community, which is frankly a bit sus.
+Nevertheless, we were excited to see DVC included in the section of Experiment
+Tracking as should! We know there are other great experiment tracking tools out
+there, and we are content to see that the larger Community is starting to
+recognize this capability with DVC! We like to think of it as taking a step
+beyond tracking to versioning. To learn more about experiment versioning,
+[visit this blog piece](https://dvc.org/blog/ml-experiment-versioning) from
+Technical Product Manager - DVC,
+[Dave Berenbaum](https://www.linkedin.com/in/david-berenbaum-20b6b424/).
+
+Our team had an internal discussion about the absence of our tools from certain
+categories, DVC and CML for artifact tracking, CML for Pipeline Orchestration
+Runtime Engine, MLEM for Model Registry and Serving. But like everything in this
+space, things are changing constantly. Thanks to whoever you are out there that
+made this nifty tool!
+
+![MyMLOps.com](/uploads/images/2022-07-15/mymlops.png '=800') _MLOps tool stack
+diagram generator from MyMLOps.com ([Source link](https://mymlops.com/))_
+
+### **Samson Zhang**: MLOps: How DVC smartly manages your data sets for training your machine learning models on top of Git
+
+[**Samson Zhang**]() of [LittleBigCode](www.littlebigcode.fr) writes an in-depth
+article in [Medium](https://medium.com) on how DVC aptly manages large datasets.
+He discusses why DVC is needed and how it is a better option compared to MLFlow
+because MLflow does not optimize storage for file duplication like DVC does, as
+well as Git-LFS for the same reasons mentioned by Alex Strick van Linschoten in
+the piece mentioned above. Samson goes through a very thorough overview of the
+tool, how it works and how to use it. He includes some best practices that he
+has figured out while using the tool and goes over how to set up a dataset
+registry which he finds particularly useful with DVC.
+
+![Samson Zhang, DVC Workflow, Cache and Storage](/uploads/images/2022-07-15/samson-zhang.png '=800')
+_DVC workflow, cache, and storage
+([Source link](https://medium.com/hub-by-littlebigcode/mlops-how-dvc-smartly-manages-your-data-sets-for-training-your-machine-learning-models-on-top-of-b73857e54e52))_
+
+### **Dror Atariah**: Getting to Know MLEM
+
+![Awesome MLEM](/uploads/images/2022-07-15/awesome.png 'Getting to Know MLEM :wrap-left =300')
```

Comment:
This appears quite large on my screen; not the best addition visually IMO.

Reply:
Yes!  I need help here.  It's supposed to wrap.  I'm applying the code the same way I have in past editions and it's not working.  I don't know why. (cc: @julieg18 )

Reply:
Seems the merge to `main` fixed this.
![image](https://user-images.githubusercontent.com/9111807/179518368-562342b5-180f-463b-8eb2-9c89e8570d15.png)
I looked into the code we use to process these declarations since it was updated relatively recently, but nothing looked off.

Reply:
Thanks @rogermparent! I ended up moving this blog post to [a new branch](https://github.com/iterative/dvc.org/pull/3774) that is set up properly with main and it's working now.